src/ui/RealmBrowser/Content/Table/Cell.tsx

```diff
@@ -50,17 +50,22 @@ const getCellContent = ({
   property: IPropertyWithName;
   value: any;
 }) => {
+  // Override EditMode for primary keys, to avoid "not editable" popup when tabbing through cells
+  if (property.isPrimaryKey) {
+    editMode = EditMode.Disabled;
+  }
```

Comment:
I would probably move this to `ContentGrid.tsx:200` where the prop is passed into the component, since otherwise the effect of `editmode` is limited for a certain subset of properties which is not obvious from the outside (consumer) of the component.

Reply:
Agreed & done.